### ISSUE TITLE
Add aria label to the current form step

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/vets-design-system-documentation#3938]
## Background
We had a finding from our 508 audit of Appoint a Representative 1.0.

> Current state of the steps is not read by JAWS.
> Defect 3 | Critical | Created:
> WCAG 1.4.1 Use of Color | Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element.
> Throughout the Site.
> Throughout the Site, the visual representation of the current state of the steps cannot be understood by assistive technology. For example, on “Steps 1 of 5 Accredited representative information,”, as a JAWS user, reading the “1 of 5”, the JAWS user cannot determine that the color-coded step 1 indicates the current step. (New: 02/24/2025).

After consulting Platform's accessibility team, the guidance to address is to add an aria label so the "1" is read as "You are on step 1" for screen readers.

<img width="1466" alt="Image" src="https://github.com/user-attachments/assets/2507e50c-dcc9-4a97-a749-cf14a190e3ca" />

## Tasks
- [ ] Add an aria label "You are on step x" to the active step (in the blue circle) on each page of Appoint a Representative

## Acceptance Criteria
- [ ] The step guidance on Appoint a Representative is accessible to all.

**Comment by caw310** [comment]
> I asked [@jeana-adhoc](https://github.com/jeana-adhoc) to test this ([@rsmithadhoc](https://github.com/rsmithadhoc) is our a11y specialist on the Design System team, but he's on vacation at the moment). Jeana found that:
> 
> "It does announce "Step 2 of 5" What it doesn't announce, and what the 508 office is saying is that it should say "You are on step 2 of 5""
> 
> We both agree this is splitting hairs and not a blocker. However, it is a simple change to make that we can make.
> 
> [@caw310](https://github.com/caw310) - Can you either transfer this issue to our repo or duplicate it so that we can fix it? Thanks!

Transferred. Will add this to a future DST sprint.

**Comment by jeana-adhoc** [comment]
I am confirming I see this fixed in Storybook and in the Accredited Rep form.

https://github.com/user-attachments/assets/880cb2ef-77a3-4e76-bbc0-571046264442
Turn on audio to hear "Step" being announced

And here's the storybook code

<img width="488" alt="Image" src="https://github.com/user-attachments/assets/d115c7ca-a51f-45a6-8087-917dde349b82" />
